cpp/daal/src/algorithms/dtrees/dtrees_train_data_helper.i

```diff
@@ -107,11 +107,22 @@ struct SplitData
     TImpurityData left;
     algorithmFPType featureValue;
     algorithmFPType impurityDecrease;
+    algorithmFPType nTotalWeight;
+    algorithmFPType nLeftWeight;
```

Comment:
Looks like you should NOT use 'n' prefix in this case

Reply:
it was fixed